docs/unified/novacustom/building-manual.md

```diff
@@ -0,0 +1,170 @@
+# Dasharo firmware building guide
+
+## Intro
+
+This guide shows how to build Dasharo firmware for NovaCustom devices.
+
+## Requirements
+
+This guide was verified on Ubuntu 22.04. In practice, any Linux distribution
+with [Docker](https://www.docker.com/) support should be enough to complete it.
+
+Make sure that you have following packages installed:
+
+* Docker
+    - follow [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
+    - follow [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
+* Git
+
+    ```bash
+    sudo apt -y install git
+    ```
+
+## Build Dasharo BIOS firmware
+
+1. Refer to the table below to set the `SDK`, `REVISION`, and `CONFIG` variables
+   based on your device and preference:
+    - using release branch will always result in the latest released version,
+    - using tag you can specify the exact version to build.
+
+| Device        | SDK version             | release branch | tag format                 | config |
+|---------------|-------------------------|----------------|----------------------------|---------------------|
+| NV4X TGL      | coreboot-sdk:0ad5fbd48d | clevo/release  | novacustom_nv4x_vX.Y.Z     | novacustom_nv4x_tgl |
```

Comment:
novacustom_nv4x_vX.Y.Z  - that tag is incorrect, it should be fixed to novacustom_nv4x_tgl_vX.Y.Z

Reply:
It depends what correct means. It is correct in a sense that in the past such tags were create and exist. `novacustom_nv4x_tgl_vX.Y.Z` tags do not exist.

Reply:
We need to add `novacustom_nv4x_tgl_vX.Y.Z` tags for all previous releases for such documentation to be correct.

Reply:
yes, this is what I mean, we made mistake in naming past tags, if we depend on those we cannot delete, but if not we can delete or maybe even rename